README.openshift.md

```diff
@@ -1,62 +1,57 @@
+1. https://github.com/openshift/kubernetes/ - which holds the actual code behind OpenShift.
 
 This split requires additional effort to introduce any API change.  The following steps
 should guide you through the process.
 
 1. The first place to introduce the changes is [openshift/api](https://github.com/openshift/api/).
-Here, you put your external API updates and when you are done run `make generate`.  If you
-need to introduce a new dependency run `make update-deps`, and almost never update `glide.yaml`
-directly.  When you're done open a PR against the aforementioned repository and ping
-[@openshift/api-review](https://github.com/orgs/openshift/teams/api-review) for a review.
+Here, you put your external API updates and when you are done run `make generate`.  If you need to
+introduce a new dependency refer to the documentation on [go
+modules](https://blog.golang.org/using-go-modules).  When you're done open a PR against the
+aforementioned repository.
 
 2. The next step includes updating the [openshift/client-go](https://github.com/openshift/client-go/)
-with the changes from step 1, since it vendors it.  To do so run `make update-deps` to pick up
-the changes from step 1 and then run `make generate` to update the client code with necessary
-changes.  When you're done open a PR against the aforementioned repository and ping
-[@openshift/sig-master](https://github.com/orgs/openshift/teams/sig-master) for a review.
-
-3. The final step happens in [openshift/origin](https://github.com/openshift/origin/) repository.
-As previously, run `make update-deps` to pick up the changes from previous two steps.  Afterwards,
-run `make update` to generate the remaining bits in origin repository. When you're done open
-a PR against the aforementioned repository and ping [@openshift/sig-master](https://github.com/orgs/openshift/teams/sig-master)
-for a review.
+with the changes from step 1, since it vendors it.  To do so run `go get
+github.com/openshift/api@master` to pick up the changes from step 1 and then run `make generate` to
+update the client code with necessary changes.  When you're done open a PR against the aforementioned
+repository.
+
+3. The next step includes updating the
+[openshift/library-go](https://github.com/openshift/library-go/) with the changes from step 1, since
+it vendors it.  To do so run `go get github.com/openshift/client-go@master` to pick up the changes
+from step 1 and then run `make update`.  When you're done open a PR against the aforementioned
+repository.
+
+4. The final step happens in the downstream fork of kubernetes, the
+[openshift/kubernetes](https://github.com/openshift/kubernetes/) repository.  As previously, run `go
+get github.com/openshift/library-go@master` to pick up the changes from previous three steps.
+Afterwards, run `make update` to generate the remaining bits in `openshift/kubernetes`
+repository. When you're done open a PR against the aforementioned repository.
```

Comment:
Why is this description here at all? How is o/k special as a consumer of library-go? I would expect this merely distributed over api, client-go and library-go repos.

Reply:
Would remove the whole list of steps here.

Reply:
Done